packages/shared/src/components/auth/AuthModalHeader.tsx

```diff
@@ -1,41 +1,31 @@
-import classNames from 'classnames';
 import React, { ReactElement } from 'react';
 import { Button } from '../buttons/Button';
 import ArrowIcon from '../icons/Arrow';
 import { CloseModalFunc } from '../modals/common';
-import { ModalCloseButton } from '../modals/ModalCloseButton';
+import { Modal } from '../modals/common/Modal';
 
 export interface AuthModalHeaderProps {
   title: string;
   className?: string;
   onBack?: CloseModalFunc;
-  onClose?: CloseModalFunc;
 }
 
 function AuthModalHeader({
   title,
   className,
   onBack,
-  onClose,
 }: AuthModalHeaderProps): ReactElement {
   return (
-    <header
-      className={classNames(
-        'flex relative flex-row items-center w-full border-b border-theme-divider-tertiary',
-        onBack ? 'p-2' : 'py-4 px-6',
-        className,
-      )}
-    >
+    <Modal.Header className={className}>
       {onBack && (
         <Button
           icon={<ArrowIcon className="-rotate-90" />}
           className="mr-2 btn-tertiary"
           onClick={onBack}
         />
       )}
-      <h3 className="font-bold typo-body">{title}</h3>
-      {onClose && <ModalCloseButton onClick={onClose} />}
```

Comment:
With this removed does it show everywhere now?
The main idea was that certain AB test don't show the close button, is that still effective?

Reply:
Yes, it shows everywhere. Should we ask product if that's ok to be consistent?

Reply:
Seems like we have ok from the product